src/plugins/english/rainofsnow.ts

```diff
@@ -0,0 +1,204 @@
+import { fetchApi } from '@libs/fetch';
+import { Plugin } from '@typings/plugin';
+import { Filters } from '@libs/filterInputs';
+import { CheerioAPI, load as parseHTML } from 'cheerio';
+import { NovelStatus } from '@libs/novelStatus';
+
+const baseUrl = 'https://rainofsnow.com/';
+
+class Rainofsnow implements Plugin.PluginBase {
```

Comment:
```suggestion
class Rainofsnow implements Plugin.PagePlugin {
```
Needs to be a PagePlugin, I'm afraid